general/g.region/testsuite/test_g_region.py

```diff
@@ -46,6 +48,48 @@ def test_f_flag(self):
         line = call_module("g.region", flags="fglecn3", capture_stdout=True)
         self.assertEqual(1, len(line.splitlines()))
 
+    def test_format_json(self):
+        """Test json format"""
+        expected = {
```

Comment:
Update expected json to match output with additional flags.

Reply:
Done.